application/modules/core/views/backend-chunk/index.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ * @var yii\web\View $this
+ * @var yii\data\ActiveDataProvider $dataProvider
+ * @var \app\models\Form $searchModel
+ */
+
+use app\backend\components\ActionColumn;
+use kartik\dynagrid\DynaGrid;
+use kartik\grid\BooleanColumn;
+use kartik\helpers\Html;
+use kartik\icons\Icon;
+
+$this->title = Yii::t('app', 'Content Blocks');
+$this->params['breadcrumbs'][] = $this->title;
+
+?>
+
+<?= app\widgets\Alert::widget([
+    'id' => 'alert',
+]); ?>
+
+<?=
+DynaGrid::widget([
+    'options' => [
+        'id' => 'properties-grid',
+    ],
+    'columns' => [
+        [
+            'class' => \kartik\grid\CheckboxColumn::className(),
+            'options' => [
+                'width' => '10px',
+            ],
+        ],
+        [
+            'class' => 'yii\grid\DataColumn',
+            'attribute' => 'id',
+        ],
+        'name',
+        'key',
+        'value',
```

Comment:
Do we really need value? Values can be too big remove it or use 'value:truncate'